docs/get_started/tour.mdx

```diff
@@ -45,1015 +51,828 @@ This guide is written for:
+            2024-04-16 17:32:10 DEBUG [CartObject/checkout][inv_1aiqX0vFEFNH7u1kZjvyH4KJpuO9j4njCp] dev.restate.sdk.core.InvocationStateMachine - Transitioning state machine to REPLAYING
+            2024-04-16 17:32:10 DEBUG [CartObject/checkout][inv_1aiqX0vFEFNH7u1kZjvyH4KJpuO9j4njCp] dev.restate.sdk.core.InvocationStateMachine - Current journal entry [1](): InvokeEntryMessage
+            // withClass highlight-line
+            2024-04-16 17:32:10 DEBUG [CheckoutService/handle] dev.restate.sdk.http.vertx.RequestHttpServerHandler - Handling request to CheckoutService/handle
+            2024-04-16 17:32:10 INFO  [CheckoutService/handle] dev.restate.sdk.core.ResolvedEndpointHandlerImpl - Start processing invocation
+            2024-04-16 17:32:10 DEBUG [CheckoutService/handle][inv_12rkfeAcppNY3cI4F6DBZK8fir8uaIrIBP] dev.restate.sdk.core.InvocationStateMachine - Transitioning state machine to REPLAYING
+            2024-04-16 17:32:11 DEBUG [CheckoutService/handle][inv_12rkfeAcppNY3cI4F6DBZK8fir8uaIrIBP] dev.restate.sdk.core.InvocationStateMachine - Current journal entry [1](): OutputEntryMessage
+            2024-04-16 17:32:11 INFO  [CheckoutService/handle][inv_12rkfeAcppNY3cI4F6DBZK8fir8uaIrIBP] dev.restate.sdk.core.InvocationStateMachine - End invocation
+            2024-04-16 17:32:11 DEBUG [CheckoutService/handle][inv_12rkfeAcppNY3cI4F6DBZK8fir8uaIrIBP] dev.restate.sdk.core.InvocationStateMachine - Transitioning state machine to CLOSED
+            2024-04-16 17:32:11 INFO  [CheckoutService/handle][inv_12rkfeAcppNY3cI4F6DBZK8fir8uaIrIBP] dev.restate.sdk.core.InvocationStateMachine - End invocation
+            2024-04-16 17:32:11 DEBUG [CartObject/checkout][inv_1aiqX0vFEFNH7u1kZjvyH4KJpuO9j4njCp] dev.restate.sdk.core.InvocationStateMachine - Current journal entry [2](): OutputEntryMessage
+            2024-04-16 17:32:11 INFO  [CartObject/checkout][inv_1aiqX0vFEFNH7u1kZjvyH4KJpuO9j4njCp] dev.restate.sdk.core.InvocationStateMachine - End invocation
+            2024-04-16 17:32:11 DEBUG [CartObject/checkout][inv_1aiqX0vFEFNH7u1kZjvyH4KJpuO9j4njCp] dev.restate.sdk.core.InvocationStateMachine - Transitioning state machine to CLOSED
+            2024-04-16 17:32:11 INFO  [CartObject/checkout][inv_1aiqX0vFEFNH7u1kZjvyH4KJpuO9j4njCp] dev.restate.sdk.core.InvocationStateMachine - End invocation
+            ```
+
+        </TabItem>
+    </Tabs>
 
-</TabItem>
+</details>
+
+## Durable Execution
+
+The calls we just did seem like regular RPC calls as you might know them from other service frameworks.
+But under the hood a lot more is happening.
+
+Restate makes RPC calls resilient by letting the Restate Server and SDK cooperate.
+Restate tracks the execution of code in a journal and can replay it in case of a failure.
+This is called **Durable Execution.**
+
+Have a look at the animation to understand what happened under-the-hood:
+
+<TourAnimation/>
```

Comment:
Animation doesn't work with arrow keys :)

Reply:
Also for someone like me that clicks compulsively, a "go back to previous step" button is more useful than reset :)

Reply:
I agree but the animation started acting weirdly with the 'previous' step, and it was taking a lot of time. So I wrote it down as a stretch goal.
Maybe I will fix it after the 0.9 release but before the launch.